torch_geometric/nn/models/pmlp.py

```diff
@@ -94,7 +94,7 @@ def forward(
                 x = x.relu()
                 x = F.dropout(x, p=self.dropout, training=self.training)
 
-        return x
```

Comment:
Can you check whether we also apply softmax in other models? IMO, it should be consistent across models.

Also, this is a breaking change, meaning it'd require a changelog entry.

Reply:
all models in this example applied softmax, and also updated the changelog